src/settings.py

```diff
@@ -4,6 +4,7 @@
 load_dotenv()
 COURSE_ID = helpers.get_course_code()
 
+INST_CODE = 112240000000000000 # WORKS FOR UBC
```

Comment:
Out of curiosity, what is this :p

I see there are areas where you subtract this value from "global" ids to get user and course ids. Is it just some institutional value that gets added to each ID to make it more secure?

Reply:
I guess so - this is what makes the values BIGINT (a data specific type) -> but a lot of tools don't do well with BIGINTs (or at least Canvas's) - like python, Tableau etc. I've seen some strange behaviour